.github/workflows/builder.yml

```
@@ -219,12 +188,9 @@ jobs:
         version: ["3.20", "3.21", "3.22"]
         python: ["3.12", "3.13", "3.14"]
         exclude:
-          - python: "3.14"
-            arch: "armv7"
-          - python: "3.14"
-            arch: "armhf"
-          - python: "3.14"
-            arch: "i386"
+          - arch: "armv7"
+          - arch: "armhf"
+          - arch: "i386"
```

```suggestion
        exclude: *matrix-exclude-build
```

_If still necessary._